### ISSUE TITLE
The performance of the code

### ISSUE THREAD

**Issue by TFLQW** [Magica-Chen/gptp_multi_output#5]
Recently I have ran the **SimulatedExample.m**, I found the result is not as good as that in your paper.
Figure 1: the result of this code ; Figure 2 : the result of the paper.
Could your tell me how to set the parameters of this package to improve the performance.
Figure 1:
![图片](https://user-images.githubusercontent.com/34564497/86866422-88a81180-c103-11ea-8874-47783a961f90.png)
Figure 2:
![图片](https://user-images.githubusercontent.com/34564497/86866482-a5dce000-c103-11ea-8970-a96d38dee7f0.png)

**Comment by Magica-Chen** [comment]
@TFLQW  Thanks for your interest. First of all, as I mentioned in the code:
```
% Example code for multi-output prediction using some simulated data (It is
% not the code for the following paper, there are some slight differences)
```
Also, you can find I mentioned in the paper:
```
The ARMSEs for both training and test points in [- 10, 10] are reported in Table 3, and an **example** of
prediction is demonstrated in Fig. 1.
``` 

This example code just shows how it works, as similar to the paper's figure. Both of them are examples, 
Since one run will highly reply to the random seed setting and other environmental variables, you need to **run many times** and compute the ARMSE (average root mean square error) or other criteria to evaluate the model performance.

**Comment by TFLQW** [comment]
@Magica-Chen Thanks for your reply, what I understand is that, the performance of MV-GPR may depend on the initial parameters setting, like the SE_init，so can you recommand some initialization function which could get a better performance?

**Comment by Magica-Chen** [comment]
@TFLQW Yeah, you're right. Initial hyperparameters may have a huge influence on hyperparameter learning and then the performance of the model.  However, in my previous paper: "How priors of initial hyperparameters affect Gaussian process regression models", I find that even hyperparameter estimation is sensitive to the initial hyperparameters, the model performance is considerable in single GPR model with SE kernel. Here 'considerable performance' means, if you run the model 200 times, and look at the average performance, initial hyperparameters of SE kernel may have less effect on the performance. 
But, to be honest, similar research on MV-GPR has not been done, so I cannot draw a conclusion about that, but I think for SE kernel, it is probably fine for the different initials.  Therefore, this is also I just use the common-used way to initialisation.
```
hyp = rand(n_parameter,1);      % This is the most commonly-used method
```

For some specific cases, maybe you can take advantage of the some real meanings of your data, like you know something about the  'lengthscale' of data, then you can use more suitable initial prior.

**Comment by TFLQW** [comment]
@Magica-Chen Thanks, I got it!
